pkg/operator/starter.go

```diff
@@ -214,10 +215,27 @@ func (o *OperatorOptions) RunOperator(ctx context.Context, controllerContext *co
 	return nil
 }
 
-func (o *OperatorOptions) getFeatureGateMappingFromDisk() (map[configv1.FeatureSet]*configv1.FeatureGateEnabledDisabled, error) {
+func (o *OperatorOptions) getFeatureGateMappingFromDisk(ctx context.Context, configClient configv1client.Interface) (map[configv1.FeatureSet]*configv1.FeatureGateEnabledDisabled, error) {
+	// TODO get the cluster profile in a better way, this isn't strictly correct
```

Comment:
How does CVO work it out? Is this the same mechanism?

Reply:
> How does CVO work it out? Is this the same mechanism?

CVO gets it via an env var that no other pod gets.